data/authors/oriononline.yaml

```diff
@@ -0,0 +1,7 @@
+name: oriononline
+bio: |
+    OrionOnline (or Orion for short) is a maintainer for NeoForged, primary maintainer of the Toolchain, as well as a member of the Porting, Triage and Rendering teams.
```

Comment:
There's no Triage or Rendering team, and the Porting team is basically synonymous with the maintainer team :P

Reply:
Their is actually a rendering team
You can ask curle